src/wallet/wallet2.cpp

```diff
@@ -872,7 +905,7 @@ bool get_pruned_tx(const rpc::GET_TRANSACTIONS::entry &entry, cryptonote::transa
     else
     {
       // for v1, we trust the dameon
-      CHECK_AND_ASSERT_MES(tools::hex_to_type(entry.tx_hash, tx_hash), false, "Failed to parse tx hash");
+      CHECK_AND_ASSERT_MES(tools::hex_to_type(entry["tx_hash"].get<std::string>(), tx_hash), false, "Failed to parse tx hash");
```

Comment:
am I going to be causing performance issues by calling `get<std::string>()` so much?

Reply:
nlohmann supports `.get<std::string_view>()`, though probably isn't worth worrying about for wallet2.